pkgs/jni/CHANGELOG.md

```diff
@@ -44,6 +44,11 @@
 - **Breaking Change**: `JArray.filled` now uses the generated type class of the
   `fill` object and not its Java runtime type.
 
+## 0.7.3
```

Comment:
How will you release a version 0.7.3? We don't currently have a `jnigen-stable` branch where you can make this PR on.

Reply:
I already did, I made a 0.7.3 branch. I can rename it to jnigen-stable as well: https://github.com/dart-lang/native/tree/jni-0.7.3

Reply:
Okay.

It's probably fine to just add a tag and delete the branch.

(We need a branch if we need to do multiple fixes on old stable releases.)